src/Illuminate/Validation/Concerns/ReplacesAttributes.php

```diff
@@ -355,4 +355,25 @@ protected function replaceAfterOrEqual($message, $attribute, $rule, $parameters)
     {
         return $this->replaceBefore($message, $attribute, $rule, $parameters);
     }
+
+    /**
+     * Replace all place-holders for the dimensions rule.
+     *
+     * @param  string  $message
+     * @param  string  $attribute
+     * @param  string  $rule
+     * @param  array   $parameters
+     * @return string
+     */
+    protected function replaceDimensions($message, $attribute, $rule, $parameters)
```

Comment:
Can be type hinting: `protected function replaceDimensions($message, $attribute, $rule, array $parameters)`